src/box/lua/schema.lua

```diff
@@ -1960,6 +1962,44 @@ end
 
 internal.check_iterator_type = check_iterator_type
 
+local function check_option_after(opts)
```

Comment:
Let's introduce `check_pairs_opts` instead that would check all options of `pairs`, similarly to `check_select_opts`.

Reply:
Done